### ISSUE TITLE
Inverse lookup of controls

### ISSUE THREAD

**Issue by weakman54** [tesselode/baton#19]
Having the ability of checking which inputs match which actions would be helpful in some situations, for example in tutorials. For example showing all the controls corresponding to "attack" with a message along the lines of "To counter an enemy attack, press attack {Show graphics corresponding to the controls attached to "attack"}

I'm pretty sure this is pretty easy to get at already by digging into the config table, but would be nice to have "official" functions for it. 

Other features I realized might be nice: Pattern matching for these too (see my other issue: #18 ) and/or a flag to only get the controls that correspond to the current active control-method (keyboard vs. controller)

**Comment by tesselode** [comment]
Is there anything you find unsatisfactory about reading the config table? Is it awkward to parse strings like `'key:return'`?

**Comment by weakman54** [comment]
Not really, I do feel like it would be nice to have an actual function for it, since right now I had to actually look into the source code to figure out how to do it, and I would also have to implement the function myself. So all in all, just a Quality of Life feature =)
